src/components/LeafletMap.js

```diff
@@ -202,6 +218,26 @@ class LeafletMap extends Component {
     });
   }
 
+  shouldComponentUpdate(nextProps, nextStates) {
```

Comment:
Suggest you throw a line or three of commenting on why you're using such a rare lifecycle element.